web/src/graphql/RealmApolloProvider.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useRealmApp } from "../RealmApp";
+import {
+  ApolloClient,
+  HttpLink,
+  InMemoryCache,
+  ApolloProvider,
+} from "@apollo/client";
+
+const createRealmApolloClient = (app) => {
```

Comment:
Would definitely be useful throwing some comments in this "createRealmApolloClient()" method since it has quite a bit of fairly complex functionality.